colobot-base/src/script/scriptfunc.cpp

```diff
@@ -72,6 +72,13 @@
 
 using namespace CBot;
 
+enum CameraView
+{
+    DefaultView,
+    FirstPersonView,
+    ThirdPersonView,
+};
+
```

Comment:
New enum types should be preferably `enum class`. Enumerators should use `SCREAMING_SNAKE_CASE`.

Reply:
Done